pom.xml

```diff
@@ -216,7 +216,7 @@
     <groupId>io.netty</groupId>
     <artifactId>netty-transport-native-epoll</artifactId>
     <classifier>linux-x86_64</classifier>
```

Comment:
@andrewazores is this classifier going to affect our multi-arch builds?

Reply:
I need to test it alongside https://github.com/cryostatio/cryostat/pull/1548 , which adds a classifier and multiarch logic that should handle it.